go/ngql/query.go

```diff
@@ -56,21 +62,27 @@ func NewRootQueryObject(rootValue types.Value, tm *TypeMap) *graphql.Object {
 // NewContext creates a new context.Context with the extra data added to it
 // that is required by ngql.
 func NewContext(vr types.ValueReader, tm *TypeMap) context.Context {
-	return context.WithValue(context.WithValue(context.Background(), vrKey, vr), tmKey, tm)
+	return context.WithValue(context.Background(), vrKey, vr)
```

Comment:
As the API is now it is a bit hard. Once I've removed the old API I can make that change.